---
abstract: Tissue-specific genes are a class of genes whose functions and expressions are preferred or restricted in one or several tissues. Identification of tissue-specific genes is essential for discovery of tissue-specific targets and understanding molecular mechanisms. Intra-tumoral heterogeneity is defined mainly as diversity among the cells and layers of only one single tumor tissue which limits therapeutic efficacy and lead to resistance to therapy. Thus, tumor heterogeneity is an important challenge to be tackled for successful personalized medicine. In this study, we aimed to identify tissue-specific genes in more rigorous fashion, to examine intersection between differentially expressed genes (DEG) in various cancers and tissue-specific genes, to analyze cancer gene expression profiles in terms of tissue specificity to demonstrate intra-tumoral heterogeneity, to interpret tumor heterogeneity, reveal tissue-specific molecular targets for different solid tumors and to elucidate roles of tissue-specific genes in biological processes. Gene expression data, derived from five large RNA-Seq projects, spanning 96 different human tissues were retrieved from ArrayExpress and ExpressionAtlas. The detection process of the tissue-specific genes not only included tau calculation as a specificity score, but also required integrating specificity index, tau score, and statistical distance as method to assign genes to multiple tissues. DEG of 16 different cancers were taken from BioExpress and we investigated intersection of cancer DEG with specifically expressed genes in corresponding tissue. After that, gene expression data for 11 different primary solid tumors, retrieved from The Cancer Genome Atlas (TCGA), were analyzed by integrating our findings about tissue-specific expression which allowed to understand tumoral heterogeneity and cancer cell behaviors. Significant genes obtained after all calculations were functionally annotated for identifying of their roles in biological processes. As a result, we successfully assigned genes to multiple tissues for specificity using a novel approach. Then discovered that DEG in a cancer tissue has low overlap with genes specific to that tissue. Finally, and most importantly, integrated our findings with TCGA gene expression data to tackle the complicated phenomenon called tumor heterogeneity. We identified genes which are candidate biomarkers for heterogeneity exhibiting gene expression in cancer tissue even though its expression is restricted to unrelated tissue. The genes identified in our study were already shown in literature to be associated with interesting phenomena, tumor heterogeneity or genetic instability of cancer cells, about various cancers. Our results are likely to contribute to the road paved by many researchers trying to understand cancer for many years by bringing the tissue-specific genes into the scene. 

authors:
- KONUK, B. H.
date: "2018-07-01T00:00:00+03:00"
doi: ""
featured: true
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)'
  focal_point: ""
links:
- name: Custom Link
  url: http://example.org
projects:
- internal-project
publication: In *YTÜ Fen Bilimleri Enstitüsü*
publication_short: In *YTÜ Fen Bilimleri Enstitüsü*
publication_types:
- "7"
slides: example
summary: .
tags:
- Biology
- Genetics
- Tissue-specific genes
- RNA-Sequencing
- Tau score
- Statistical distance
- Tumor heterogeneity
- Genetic instability of cancer cell

title: Analysis of Intra-tumoral Heterogeneity in Context of Tissue Specific Gene Expression With Computational Approach
url_code: '#'
url_dataset: '#'
url_pdf: http://eprints.soton.ac.uk/352095/1/Cushen-IMV2013.pdf
url_poster: '#'
url_project: ""
url_slides: ""
url_source: '#'
url_video: '#'
---

{{% alert note %}}
Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
{{% /alert %}}
